client/blocks/app-banner/style.scss

```diff
@@ -91,10 +92,14 @@ body.app-banner-is-visible {
 
 	&.jetpack {
 		font-size: rem(28px); //typography-exception
-		font-weight: bold;
+		font-weight: 500;
 		max-width: 18em;
 		margin-left: auto;
 		margin-right: auto;
+		-webkit-font-smoothing: auto;
+		-moz-osx-font-smoothing: initial;
+		text-rendering: auto;
```

Comment:
'Reverting' the following font-smoothing CSS, without this the font-weight doesn't appear to be `500`:  

```
smoothing-antialiased {
    text-rendering: optimizeLegibility;
    -moz-osx-font-smoothing: grayscale;
    -webkit-font-smoothing: antialiased;
}
```